### PR TITLE
Added logic to restart the game when the ship dies

### LINKED ISSUE #150: Add Logic to restart the game

### PR DESCRIPTION
**Pull request by junior0123**

### CODE REVIEW

**Review by RodrigoValda — DISMISSED** [review]
Nice job, good test cases

**Review by RodrigoValda — APPROVED** [review]
Good job with the bullet class, we have to advance the unit tests

File: src/main/java/jalau/at18/spaceinvaders/Game.java [hunk 1]
```diff
@@ -1,55 +1,80 @@
...
-                    board.setElement(ship.getPosY(), ship.getPosX() + 1, '*');
-                    // System.out.println(ship.getPosX());
-                } else if (event.getKeyCode() == KeyEvent.VK_RIGHT && ship.getPosX() < SIZE_BOARD) {
-                    ship.moveRight();
-                    board.setElement(ship.getPosY(), ship.getPosX(), '^');
-                    board.setElement(ship.getPosY(), ship.getPosX() - 1, '*');
-                } else if (event.getKeyCode() == KeyEvent.VK_SPACE) {
-
-                    System.out.println("fire");
+        for (int lifes = 0; lifes < LIFES_SHIP; lifes++) {
```
Comment (TolabaLib) [inline src/main/java/jalau/at18/spaceinvaders/Game.java:25]: Maybe LIFES_SHIP could be ship.getLifes

File: src/main/java/jalau/at18/spaceinvaders/Bullet.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package jalau.at18.spaceinvaders;
```
Comment (TolabaLib) [inline src/main/java/jalau/at18/spaceinvaders/Bullet.java:1]: This class should be abstract to work on the movement of the ship's bullet and aliens separately

**Pull request merged**